code/parse.go

```diff
@@ -99,6 +101,10 @@ func parseCloudWatchLogs(request events.CloudwatchLogsEvent) []ingest.Log {
 		result := re1.FindStringSubmatch(d.LogGroup)
 		lambdaName := result[1]
 		arn = fmt.Sprintf("arn:aws:lambda:%s:%s:function:%s", awsRegion, d.Owner, lambdaName)
+	} else if strings.Contains(d.LogGroup, "/aws/ec2/networkInterface") {
+		splitLogStream := strings.Split(d.LogStream, "-")
+		arn = splitLogStream[0] + "-" + splitLogStream[1]
+		resourceID = "system.aws.networkInterfaceIds"
```

Comment:
If we are not sending the arn for network interface. Maybe we should rename the variable to something else.
I am curious, if we could still do it with arn. AWS should be able to identify that.

Reply:
I don't think we would be able to do with ARN as EC2 recognises network interface through a property. Please refer to comment section of https://jira.logicmonitor.com/browse/DEV-68228.

I will update the variable name.